lib/commands/new/project-template.js

```diff
@@ -28,6 +28,8 @@ exports.ProjectTemplate = class {
       paths: {}
     });
 
+    this.model.serve = this.createServeConfig();
```

Comment:
I think we should put the port and logLevel in the `platform` object, similar to `baseDir`